test/support/AuthStub.scala

```diff
@@ -105,4 +105,65 @@ object AuthStub {
                       ]
              }
        """.stripMargin)))
+
+  def authOkWithSeveralEnrolments(vrnList: List[Vrn], enrolments: List[String]): StubMapping =
+    stubFor(post(urlEqualTo("/auth/authorise"))
+      .willReturn(aResponse()
+        .withStatus(200)
+        .withBody(
+          s"""
+             {
+               "new-session":"/auth/oid/$oid/session",
+               "enrolments":"/auth/oid/$oid/enrolments",
+               "uri":"/auth/oid/$oid",
+               "loggedInAt":"2016-06-20T10:44:29.634Z",
+               "optionalCredentials":{
+                 "providerId": "12345",
+                 "providerType": "GovernmentGateway"
+               },
+               "accounts":{
+               },
+               "lastUpdated":"2016-06-20T10:44:29.634Z",
+               "credentialStrength":"strong",
+               "confidenceLevel":50,
+               "userDetailsLink":"$wireMockBaseUrlAsString/user-details/id/$oid",
+               "levelOfAssurance":"1",
+               "previouslyLoggedInAt":"2016-06-20T09:48:37.112Z",
+               "groupIdentifier": "groupId",
+               "affinityGroup": "Individual",
+               "allEnrolments": [
+                        {
+                          "key": "${enrolments.head}",
```

Comment:
`enrolments` is a general `List` in this method so you have no guarantee that there'll be a head or if there'll be three things in total. What would be safer is map each thing in the `List` into a Json string representing each object in the array and then doing `.mkString(",")` on it. To make this easier maybe having one `List[(Vrn, String)]` would be easier than passing in two separate lists so you don't have to worry about if the lists are somehow different sizes